rgw/v2/lib/sync_status.py

```diff
@@ -0,0 +1,51 @@
+import os, sys, glob
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../")))
+from v2.lib.resource_op import Config
+import time
+import v2.utils.utils as utils
+from v2.lib.exceptions import SyncFailedError
+import logging
+
+log = logging.getLogger()
+
+def sync_status(retry=5,delay=30):
+    """
+    verify multisite sync status
+    """
+    log.info('check sync status')
+    cmd = ('sudo radosgw-admin sync status')
+    check_sync_status = utils.exec_shell_cmd(cmd)
+
+    #check for 'failed' or 'ERROR' in sync status. 
+    if "failed|ERROR" in check_sync_status:
+        log.info('checking for any sync error')
+        cmd = 'sudo radosgw-admin sync error list'
+        sync_error_list = utils.exec_shell_cmd(cmd)
+        raise SyncFailedError("sync status is in failed or errored state!")    
+    else:
+        log.info('No errors or failures in sync status')
+
+    #check if sync is in progress, if sync is in progress retry 5 times with 30 secs of sleep between each retry
```

Comment:
log this and also since delay and retry are params have default values, so use the variables here when you log

Reply:
Sure Rakesh, added the variables in the log.